hopp/eco/finance.py

```diff
@@ -378,7 +378,8 @@ def run_profast_lcoe(
         },
     )
     pf.set_params(
-        "capacity", hopp_results["annual_energies"]["wind"] / 365.0
+        "capacity", sum(hopp_results['combined_power_production_hopp'])/365
```

Comment:
let's keep the decimal in the denominator in case somehow the numerator is an int.